src/ORM/FieldType/DBBoolean.php

```diff
@@ -62,13 +62,13 @@ public function scaffoldSearchField($title = null)
     {
         $anyText = _t(__CLASS__ . '.ANY', 'Any');
         $source = [
+            '' => $anyText,
             1 => _t(__CLASS__ . '.YESANSWER', 'Yes'),
             0 => _t(__CLASS__ . '.NOANSWER', 'No')
         ];
 
-        $field = new DropdownField($this->name, $title, $source);
-        $field->setEmptyString("($anyText)");
-        return $field;
+        return (new DropdownField($this->name, $title, $source))
```

Comment:
Would using the injector be better instead of instantiating the class directly?

Reply:
It would be. Unfortunately, most DB fields do still circumvent Injector in their `scaffoldSearchFields()` implementations. I think it's just code from 3.x that survived the upgrade in tact.

Reply:
Yes, it would...  Do you mean:

```php
DropdownField::create()
```

Happy to change it.

Reply:
I'd say let's do that in a separate PR that applies injector to all these core implementations, rather than just do a one-off. The problem is, if we have some fields that are responding to Injector overrides and some that aren't, it just creates a confusing dev experience. In my quick assessment, it looks like most fields are still using `new`, so I'd advocate for just going with the pack, here. 

Consistent use of Injector in a separate body of work would be sweet, though.